lib/canary/plugs.ex

```diff
@@ -325,6 +353,10 @@ defmodule Canary.Plugs do
     end
   end
 
+  defp get_persisted(opts) do
```

Comment:
This is purely stylistic, but given how it is used, (as boolean), I think this function would be better named to follow the convention for functions that return booleans, e.g. `def persisted?`.

Then when you use it, you can write `and not persisted?(opts)` rather than `and not get_persisted(opts)`